README.md

```diff
@@ -44,6 +44,7 @@ jobs:
       - uses: k2bd/advent-readme-stars@v1
         with:
           userId: 1234567
+          leaderboardId: 9876543
```

Comment:
Nit: in the lines above, could the text "pasting in yout user ID instead of 1234567:" be replaced with "pasting in your user ID and optional leaderboard ID"?